docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md

```diff
@@ -19,44 +19,57 @@ Whatever your application's needs and architecture, Windows Azure can support it
 
 ### App Service Web Apps
 
 App Service Web Apps offers a fully managed platform optimized for hosting web applications. It is a platform-as-a-service(PaaS) offering that lets you focus on your business logic, while Azure takes care of the infrastructure needed to run and scale the app. Some key features of App Service Web Apps:
 
--   DevOps optimization (continuous integration and delivery, multiple environments, A/B testing, scripting support)
+- DevOps optimization (continuous integration and delivery, multiple environments, A/B testing, scripting support)
 
--   Global scale and high availability
+- Global scale and high availability
 
--   Connections to SaaS platforms and your on-premises data
+- Connections to SaaS platforms and your on-premises data
 
--   Security and compliance
+- Security and compliance
 
--   Visual Studio integration
+- Visual Studio integration
 
 Azure App Service is the best choice for most web apps. Deployment and management are integrated into the platform, sites can scale quickly to handle high traffic loads, and the built-in load balancing and traffic manager provide high availability. You can move existing sites to Azure App Service easily with an online migration tool, use an open-source app from the Web Application Gallery, or create a new site using the framework and tools of your choice. The WebJobs feature makes it easy to add background job processing to your App Service web app.
 
 ### Containers and Azure Container Service
 
-Azure Container Service makes it simpler for you to create, configure, and manage a cluster of virtual machines that are preconfigured to run containerized applications. It uses an optimized configuration of popular open-source scheduling and orchestration tools. This enables you to use your existing skills, or draw upon a large and growing body of community expertise, to deploy and manage container-based applications on Microsoft Azure.
-
-Azure Container Service makes it simpler for you to create, configure, and manage a cluster of virtual machines that are preconfigured to run containerized applications. It uses an optimized configuration of popular open-source scheduling and orchestration tools. This enables you to use your existing skills, or draw upon a large and growing body of community expertise, to deploy and manage container-based applications on Microsoft Azure.
+Azure Container Service (AKS) makes it simpler for you to create, configure, and manage a cluster of virtual machines that are preconfigured to run containerized applications. It uses an optimized configuration of popular open-source scheduling and orchestration tools. This enables you to use your existing skills, or draw upon a large and growing body of community expertise, to deploy and manage container-based applications on Microsoft Azure.
```

Comment:
Azure Container Service has been replaced by AKS - Azure Kubernetes Service. We should only be using Azure Container Service if you're referring to the older product.

Reply:
Azure Kubernetes Service is described in the following paragraph. Should we just remove the paragraph about "Containers and Azure Container Service" ? I'm going to go ahead and just do that.

Reply:
I think it's the right call.